### DISCUSSION TITLE
CSV catalog item ideal zoom disabled

### DISCUSSION THREAD

**Discussion by gtgrp-user** [TerriaJS/terriajs#6044] [Q&A] [answered]
Hi @nf-s ,

Is there a reason why the CSV catalog ideal zoom is disabled? Occasionally, loading ideal zoom gets stuck, though the data is loaded. In the browser console there are a few error messages:

`Failed to load resource: the server responded with a status of 404 ()
vector-tiles.terria.io/POA_2016_AUST/8/146/135.pbf:1 Failed to load resource: the server responded with a status of 404 ()
vector-tiles.terria.io/POA_2016_AUST/8/146/136.pbf:1 Failed to load resource: the server responded with a status of 404 ()
vector-tiles.terria.io/POA_2016_AUST/8/147/135.pbf:1 Failed to load resource: the server responded with a status of 404 ()
vector-tiles.terria.io/POA_2016_AUST/8/145/135.pbf:1 Failed to load resource: the server responded with a status of 404 ()
vector-tiles.terria.io/POA_2016_AUST/8/145/136.pbf:1 Failed to load resource: the server responded with a status of 404 ()`

Many thanks.

Pritimoy

**Comment by nf-s** [comment, marked as answer]
Hi Pritimoy,

CSVs which use region mapping do not have ideal zoom enabled at the moment.
Because we use vector tiles for regions, it is quite hard to calculate the ideal zoom for these layers - https://github.com/TerriaJS/terriajs/issues/920
It isn't on our timeline at the moment, but it would be really nice to have!

We also have a bug in Leaflet/2D mode when you drag and drop CSVs into Terria - not sure if this is relevant to you - but it does cause the ideal zoom to hang - https://github.com/TerriaJS/terriajs/issues/6040

Also, do you mind if I turn this issue into a discussion?

Cheers,
Nick

**Comment by gtgrp-user** [comment]
Hi Nick,

Thanks for your reply. 

Bug in Leaflet/2D mode is relevant to me and feel free to turn this issue into a discussion. 

Pritimoy
